### DISCUSSION TITLE
Support edge-to-edge rendering on Android

### DISCUSSION THREAD

**Discussion by Seanathan10** [community/community#59369] [Mobile] [answered]
Since the introduction of gesture navigation with Android 10, apps can render below the status bar and navigation bar for a cleaner look.

![Screenshot_20230626-120547.png](https://github.com/community/community/assets/38306929/60a09fae-4cf4-4b90-86eb-db4ce3e957fd)

This is not only wasted space as there could be an additional line of content there, but it also gives the GitHub mobile app a worse look.

**Comment by stevepopovich** [comment, marked as answer]
HI @Seanathan10, thanks for the feedback. I passed this along to the team for prioritization.
